### PR TITLE
review changelog with latest commits regarding rpm, security issues,etc.

### PR DESCRIPTION
**Pull request by jordimassaguerpla**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,20 @@
 ## Upcoming Version
 
+- Review requirements and provides in the RPM
+PR [#277](https://github.com/SUSE/Portus/pull/277),
+PR [#278](https://github.com/SUSE/Portus/pull/278),
+PR [#280](https://github.com/SUSE/Portus/pull/280),
+PR [#273](https://github.com/SUSE/Portus/pull/273),
+- Add configure scripts for the RPM and use environment variables for
+production. See:
+https://github.com/SUSE/Portus/pull/299
```
Comment (mssola) [inline CHANGELOG.md]: PR [#299]()

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,20 @@
...
+https://github.com/SUSE/Portus/pull/298/files
```
Comment (mssola) [inline CHANGELOG.md]: Similar to above. Also, don't use the /files url

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,20 @@
...
+https://github.com/SUSE/Portus/pull/281
```
Comment (mssola) [inline CHANGELOG.md]: See above

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,20 @@
...
+- Check run time requirements like ssl, secrets. See 
+  PR [#297](https://github.com/SUSE/Portus/pull/297)
+https://github.com/SUSE/Portus/pull/286
```
Comment (mssola) [inline CHANGELOG.md]: see above

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,20 @@
...
+- Update uglifier gem for fixing a security issue (OSVDB-126747)
+https://github.com/SUSE/Portus/pull/292
```
Comment (mssola) [inline CHANGELOG.md]: see above

**Comment by jordimassaguerpla** [comment]
rebased with master and fixed the issues mssola commented.

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,7 +1,22 @@
...
+- Set admin user from a rake task and disable first-user is admin. See PR [#314]
   (https://github.com/SUSE/Portus/pull/314)
+- Review requirements and provides in the RPM
+PR [#277](https://github.com/SUSE/Portus/pull/277),
+PR [#278](https://github.com/SUSE/Portus/pull/278),
+PR [#280](https://github.com/SUSE/Portus/pull/280),
+PR [#273](https://github.com/SUSE/Portus/pull/273),
+- Add configure scripts for the RPM and use environment variables for
+production. See:
+PR [#299](https://github.com/SUSE/Portus/pull/299)
```
Comment (mssola) [inline CHANGELOG.md]: add commas

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,7 +1,22 @@
...
+PR [#298](https://github.com/SUSE/Portus/pull/298)
+PR [#281](https://github.com/SUSE/Portus/pull/281)
+- Check run time requirements like ssl, secrets. See
+PR [#297](https://github.com/SUSE/Portus/pull/297)
+https://github.com/SUSE/Portus/pull/286
```
Comment (mssola) [inline CHANGELOG.md]: PR \[ .. \]

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,7 +1,22 @@
...
+- Update uglifier gem for fixing a security issue (OSVDB-126747)
+https://github.com/SUSE/Portus/pull/292
```
Comment (mssola) [inline CHANGELOG.md]: Same as above

**Pull request merged**